### ISSUE TITLE
no commit some important files

### ISSUE THREAD

**Issue by alexliyang** [zZTaccount/retinaface-tensorflow#1]
thx for your coding,

but when I read the codes, I find that no rcnn.model files and some functions like that in train.py:
get_sym_conv,get_out,stride_concat,get_loss. 
can you upload them?

**Comment by zZTaccount** [comment]
Sorry I missed that, the code is completed now. BTW, if you want to begin training, you need resnet_v2_50.ckpt as pretrained backbone net, which is too big to upload, you can download it in [https://github.com/MachineLP/models/tree/master/research/slim](url)

**Comment by alexliyang** [comment]
OK, Thank you very much, I will try.

**Comment by alexliyang** [comment]
I train the model from scratch ,
from the print logs, some [nan]s are print , are they OK?
like the following:

INFO: print once per batch! one batch precision= [nan][0.650520802]
Debug message: precision=[3][0 1][192 1][nan][0.890625]

**Comment by zZTaccount** [comment]
[nan] is ok. How is your training going?

**Comment by alexliyang** [comment]
Oh, I use shufflenet to train , but , the result is bad.   
bbox is better than landmark. 
another thing ： losses not use tf.reduce_mean() ? is it OK?
and ,do you write test code and pb covnert code?

besides, maybe train code not use reg_loss, you can add it .

**Comment by zZTaccount** [comment]
I used to use reduce_mean, it doesn't seem to have any difference, but you can change to reduce_mean.
the test_time.py and test_widerface.py are test code, and sorry I don't have pb convert code.
I have added reg_loss in the train.py: total_loss = myloss+0.5*reg_loss.

**Comment by zZTaccount** [comment]
This is my QQ: 429834658, maybe we can communicate through it for convenience.
